A32NX/html_ui/Pages/VCockpit/Instruments/A320_Neo/Map/Svg/A32NX_SvgMaskElement.js

```diff
@@ -55,11 +55,12 @@ class SvgPlanMaskElement extends SvgMaskElement {
             this.path.setAttribute("y", "0");
             this.path.setAttribute("fill", "url(#Backlight)");
             this.path.setAttribute("transform", "translate(" + offsetX + " " + offsetY + ")");
-            const d = "M 0,0 V 1000 H 1000 V 0 Z m 500,282.07812 c 44.58849,0.034 88.09441,13.74154 124.64648,39.27735 H 774.50781 V 778.27148 H 225.49219 V 321.35547 H 375.64062 C 412.1126,295.87541 455.50922,282.16887 500,282.07812 Z";
```

Comment:
You cannot change the mask like this. It also masks map elements such as waypoints, airports, traffic aircraft etc.

Reply:
True, I'll be looking into this more... The default mask is also wrong from the real airbus however, so this might as well be looked into while I am here.

![image](https://cdn.discordapp.com/attachments/758149448399847424/767949363787857930/iu.png)

Reply:
Ok scratch that, I think I'll just open a new issue for this. I'll try not to break it even more with the mask changes.